www/src/pages/en/usage/next-auth.mdx

```diff
@@ -39,27 +46,33 @@ const User = () => {
 
 ## Retrieving session server-side
```

Comment:
maybe this should be split to 2 sections (pages and app)

the old pages setup is still relevant afaik?

Reply:
I'm not sure i understand? The pages setup is still very relevant but i thought a warning might be a good call. In theory, the server-side retrieval section serves as the "other section" that explains how auth works in app router situations.

Reply:
you removed the gSSP part of the docs explaining how to retrieve session on the server in pages router.

Reply:
I understand what you mean now.